python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -297,7 +299,6 @@ def run(self, skyInfo, tempExpRefList, imageScalerList, weightList,
             The weight to give each input exposure in the coadd
         supplementaryData : `lsst.pipe.base.Struct`
             Result struct returned by ``makeSupplementaryData`` with components:
-
```

Comment:
I think this white space is correct, see [here](https://developer.lsst.io/python/numpydoc.html#struct-return-types)